src/release/program-github-release-assets.ts

```diff
@@ -0,0 +1,50 @@
+import { program } from 'commander'
+import fs from 'node:fs'
+import { authenticate } from '../github'
+
+type Options = {
+  deliverinoPrivateKey: string
+  owner: string
+  repo: string
+  releaseId: number
+  files: string
+}
+
+const uploadAssets = async ({ deliverinoPrivateKey, owner, repo, releaseId, files }: Options) => {
+  const appOctokit = await authenticate({ deliverinoPrivateKey, owner, repo })
+
+  for (const file of files.split('\n')) {
```

Comment:
Why have you changed this from "forEach to for", i prefer "forEach".
Also you removed the filter of "e2e", I think this should be readded, as we want to use the toolbelt for Integreat in the future too.

Reply:
Because the litter suggested it

Reply:
yes its my intelliJ linter then probably

Reply:
<img width="753" alt="image" src="https://github.com/digitalfabrik/app-toolbelt/assets/37902063/a9aecc25-f488-4e63-9f40-75602f82cdb8">

Reply:
alright i revert it